pyccel/codegen/printing/fcode.py

```diff
@@ -285,7 +322,13 @@ def print_kind(self, expr):
         """
         Prints the kind(precision) of a literal value
         """
-        return iso_c_binding[self._print(expr.dtype)][expr.precision]
+        if self._print(expr.dtype) == 'complex':
+            return iso_c_binding[self._print(expr.dtype)][expr.precision]
```

Comment:
Is it necessary to treat complexes differently? Is it not possible to have entries for complexes in `iso_c_binding` but not in `iso_c_binding_shortcut`?

Reply:
i will refactor the code to avoid treating them differently!